tensorflow/core/util/gpu_device_functions.h

```diff
@@ -125,7 +125,7 @@ __device__ inline unsigned GpuLaneId() {
   asm("mov.u32 %0, %%laneid;" : "=r"(lane_id));
 #elif TENSORFLOW_USE_ROCM
   // ROCM TODO add ROCM implementation
-  lane_id = hc::__lane_id();
```

Comment:
@yxsamliu are you able to verify if this change works with the incumbent hip/hcc toolchain?

Reply:
It builds with hip/hcc.

Which tests should I run to verify?